api/main.py

```diff
@@ -0,0 +1,122 @@
+import os
```

Comment:
Oh yeah, I had to move it to a separate file to get the auto reload working. Not sure why but the uvicorn command to run the server needs to be called with a string not an import for it to work and it didn't work in the file it was in. The reason I moved it was so that the run command from the command line would still be the same (running server.py) but the behaviour would include auto reloading.

I'm happy to refactor another way, but this seemed to work as an initial solution.